drush/Commands/DeployCommands.php

```diff
@@ -22,6 +24,13 @@ class DeployCommands extends DrushCommands implements SiteAliasManagerAwareInter
 
   use SiteAliasManagerAwareTrait;
 
+  /**
+   * Set the PHP version to use when deploying to Acquia environments.
+   *
+   * @const string
+   */
+  public const PHP_VERSION = '7.4';
```

Comment:
This is the one thing I'm not sure about. I'm a little worried that in the future this constant could be lost. I'd thought about putting it in a separate YAML file somewhere, but that adds complexity.